docs/jsonapi-spec.md

```diff
@@ -401,6 +401,23 @@ The maximum size of field values are:
 The maximum length of an array is 100 elements.
 
 
+### Equality handling with arrays and subdocs
+
+Given the query `{'foo' : "bar"}`, this matches: 
```

Comment:
the foo is enclosed with single quote and value bar is enclosed with double quote, is it done for some reason? if not can this be changed to be in double quote?

Reply:
Not done intentionally.  I think they can each use double quote... will do.

Reply:
Updated with latest commit.